docs/kratos/self-service/flows/verify-email-account-activation.mdx

```diff
@@ -183,408 +110,129 @@ your Identity Schema:
-    "method": "POST",
-    "nodes": [
-      {
-        "type": "input",
-        "group": "default",
-        "attributes": {
-          "name": "csrf_token",
-          "type": "hidden",
-          "value": "",
-          "required": true,
-          "disabled": false
-        },
-        "messages": [],
-        "meta": {}
-      },
-      {
-        "type": "input",
-        "group": "link",
-        "attributes": {
-          "name": "email",
-          "type": "email",
-          "required": true,
-          "disabled": false
-        },
-        "messages": [],
-        "meta": {}
-      },
-      {
-        "type": "input",
-        "group": "link",
-        "attributes": {
-          "name": "method",
-          "type": "submit",
-          "value": "link",
-          "disabled": false
-        },
-        "messages": [],
-        "meta": {
-          "label": {
-            "id": 1070005,
-            "text": "Submit",
-            "type": "info"
-          }
-        }
-      }
-    ]
-  },
-  "state": "choose_method"
-}
+```diff
+selfservice:
+  flows:
+    login:
++      after:
++        hooks:
++        - hook: require_verified_address
 ```
 
-## Verification Flow Payloads
-
-Fetching the Verification Flow ([REST API Reference](../../reference/api.mdx#get-verification-flow)) is usually only required for
-browser clients but also works for Verification Flows initialized by API clients. All you need is a valid flow ID:
-
-```sh
-flowId=$(curl -s -X GET \
-    -H "Accept: application/json" \
-    https://playground.projects.oryapis.com/self-service/verification/api | jq -r '.id')
-
-curl -s -X GET \
-    -H "Accept: application/json" \
-    "https://playground.projects.oryapis.com/self-service/verification/flows?id=$flowId" | jq
-
-{
-  "id": "979d00d7-4d45-4960-8566-730e37dc0a3e",
-  "type": "api",
-  "expires_at": "2021-07-09T14:11:53.197917Z",
-  "issued_at": "2021-07-09T13:11:53.197917Z",
-  "request_url": "http://playground.projects.oryapis.com/self-service/verification/api",
-  "ui": {
-    "action": "https://playground.projects.oryapis.com/self-service/verification?flow=979d00d7-4d45-4960-8566-730e37dc0a3e",
-    "method": "POST",
-    "nodes": [ /* ... */ ]
-  },
-  "state": "choose_method"
-}
-```
+For more information see the [hooks configuration](../../hooks/01_configure-hooks.mdx) documentation.
 
-### Send verification link to email
+```mdx-code-block
+  </TabItem>
+</Tabs>
+```
 
 :::note
 
-The `link` verification mode will always open a link in the browser, even if it was initiated by an API client. This is because
-the user clicks the link in his/her email client, which opens the browser.
+Please be aware, that when requiring a verified address for login, a typo in the email address set by the user either during the
+registration or when changing their email address, will make the login for that user impossible.
```

Comment:
I wanted to re-write it but a question came to my mind - is this really a problem? 
I mean, I created an account for `tmoek@email.com` and I must verify it before I do anything. 
I can't verify bc my mail is `tomek@email.com`. 
I just create a new account for a mail w/o typo and verify. 
Am I missing anything?

Reply:
Eh, I guess you're right. This was in the old doc, but it should be implied. I'll remove the note.